### PR TITLE
RELEASE: update docs for new zipfile support!

### PR DESCRIPTION
**Pull request by ctb**
@bluegenes should confirm this is how things work 😆 

Includes new behavior since 0.7.1, in PRs #99 and #105, and soon #106.

This also bumps pyo3_branchwater version to 0.8.0.

Breakdown of PRs is here: https://github.com/sourmash-bio/pyo3_branchwater/issues/108

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -105,7 +122,7 @@ A complete example Snakefile implementing the above workflow is available [in th
 
 ### Running `fastmultigather`
 
-`fastmultigather` takes a file list of query metagenomes and a file list for the database, and outputs many CSVs:
+`fastmultigather` takes a collection of query metagenomes collection of sketches as a database, and outputs many CSVs:
```
Comment (bluegenes) [inline doc/README.md:125]:
```suggestion
`fastmultigather` takes a collection of query metagenomes and a collection of sketches as a database, and outputs many CSVs:
```

File: doc/README.md [hunk 2]
```diff
@@ -4,11 +4,28 @@ This repository implements five sourmash plugins, `manysketch`, `fastgather`, `f
...
+
+If the command loads a collection of sketches into memory at the start, then the sketches from the zip file are simply loaded into memory! So,
+* `multisearch` loads both query and database into memory;
+* `manysearch` loads the queries into memory;
+* `fastgather` and `fastmultigather` both load the database into memory;
+
+If the command loads a collection of sketches throughout execution, then the zip file is _unpacked_ to a temporary directory and the sketches are loaded from there. (This can consume a lot of extra disk space!) So,
+* `manysearch` loads the sketches being searched this way;
+* `fastmultigather` loads the query sketches this way;
+
```
Comment (bluegenes) [inline doc/README.md:25]: * `fastgather` loads the sketches being searched this way

Comment (bluegenes) [inline doc/README.md:25]:
```suggestion
* `fastgather` loads the sketches being searched this way

```

**Review by bluegenes — APPROVED** [review]
other than adding the note about `fastgather` (see my suggestion) - lgtm!

**Comment by bluegenes** [comment]
might be worth adding zip info to README too, here or elsewhere..

File: doc/README.md (hunk 2 above)
Comment (ctb) [reply doc/README.md:25]: ok!

**Comment by ctb** [comment]
take a look & let me know what you think, @bluegenes! The zip additions make it much shorter 😆

**Comment by bluegenes** [comment]
looks good to me!

**Pull request merged**